src/toxcore/binary_io.rs

```diff
@@ -37,9 +37,9 @@ pub trait FromBytes<Output> {
 pub fn array_to_u16(array: &[u8; 2]) -> u16 {
     trace!("Casting array to u16 from array: {:?}", array);
     let mut result: u16 = 0;
-    for byte in 0..array.len() {
+    for pos in 0..array.len() {
```

Comment:
Well, I used `pos` and `byte` whichever one seemed to express associated meaning better in given context.

I don't really have an opinion on the matter, and given that I'm the one who wrote the code it seems to me "obvious", which may not be the case for people reading the code, so I was wondering if that would ease reading the code by someone who didn't write it. I.e. change in the name of consistency is nice & stuff, but it's preferable if there are other benefits with no downsides to it.